service/src/test/java/uk/gov/hmcts/reform/fpl/validation/validators/time/HasEndDateAfterStartDateValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.fpl.validation.validators.time;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.validation.ConstraintViolation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+class HasEndDateAfterStartDateValidatorTest extends TimeValidatorTest {
+
+    @Test
+    void shouldReturnAnErrorWhenStartDateIsAfterEndDate() {
+        hearingBooking = HearingBooking.builder()
+            .startDate(FUTURE)
+            .endDate(LocalDateTime.now().plusDays(1))
+            .build();
+
+        final List<String> violations = validator.validate(hearingBooking, group)
+            .stream()
+            .map(ConstraintViolation::getMessage)
+            .collect(Collectors.toList());
+
+        assertThat(violations).hasSize(1).containsOnlyOnce("The start date cannot be after the end date");
+    }
+
+    @Test
+    void shouldReturnAnErrorWhenDatesAreTheSame() {
```

Comment:
I think the test name is confusing - I guess the date xcan be thee same, but time cant't? I can have hearing at 3, finishing at 4. Could potentially test that.